Source/MQTTnet/Channel/IMqttChannel.cs

```diff
@@ -23,4 +24,6 @@ public interface IMqttChannel : IDisposable
     Task<int> ReadAsync(byte[] buffer, int offset, int count, CancellationToken cancellationToken);
 
     Task WriteAsync(ArraySegment<byte> buffer, bool isEndOfPacket, CancellationToken cancellationToken);
+
+    Task WriteAsync(ReadOnlySequence<byte> buffer, bool isEndOfPacket, CancellationToken cancellationToken);
```

Comment:
Do we need both methods? Is there a way to get rid of the ArraySegment based overload?

Reply:
I got rid of one method